src/task.ts

```diff
@@ -6,8 +6,21 @@ export type Quest<T extends Task = Task> = {
   tasks: T[];
 };
 
+export type OutfitSlot =
+  | "hat"
+  | "back"
+  | "weapon"
+  | "offhand"
```

Comment:
I would use `"off-hand"` for potential future compatibility with kolmafia-js's SlotType